Cargo.toml

```
@@ -1,13 +1,14 @@
 [package]
 name = "smol_str"
-version = "0.2.1"
+version = "0.2.2"
 description = "small-string optimized string type with O(1) clone"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rust-analyzer/smol_str"
 authors = ["Aleksey Kladov <aleksey.kladov@gmail.com>"]
 edition = "2018"
 
 [dependencies]
+borsh = { version = "1.4.0", optional = true}
```

This has an std feature, so
```suggestion
borsh = { version = "1.4.0", optional = true, default_features = false }
```
+ adding `"borsh?/std"` to our std feature below